data/brands/amenity/fast_food.json

```
@@ -1185,6 +1185,20 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Fairwood",
+      "id": "fairwood-7e2e93",
+      "locationSet": {"include": ["cn", "hk"]},
+      "tags": {
+        "amenity": "fast_food",
+        "brand": "Fairwood",
+        "brand:wikidata": "Q5430935",
+        "brand:wikipedia": "en:Fairwood (restaurant)",
+        "cuisine": "chinese and american",
```

The correct way to have multiple cuisines in OpenStreetMap is to separate them with a semicolon..
```suggestion
        "cuisine": "chinese;american",
```

---

Thanks for letting me know! I have updated this one and will take care of it in the future!